src/components/news/create-news/NewsPreview.js

```
@@ -0,0 +1,125 @@
+              <span>𝕏</span>
+            </button>
+            <button className="social-icon linkedin" title={t('news.create.shareLinkedIn') || 'Share on LinkedIn'}>
+              <span>in</span>
+            </button>
+            <button className="social-icon facebook" title={t('news.create.shareFacebook') || 'Share on Facebook'}>
+              <span>f</span>
+            </button>
+          </div>
+        </div>
+
+        <div className="preview-text" dangerouslySetInnerHTML={{ __html: text }}></div>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)